commons-numbers-complex/src/main/java/org/apache/commons/numbers/complex/ComplexFunctions.java

```diff
@@ -58,21 +58,112 @@
  */
 public final class ComplexFunctions {
 
-    /** Mask to remove the sign bit from a long. */
-    static final long UNSIGN_MASK = 0x7fff_ffff_ffff_ffffL;
-
+    /** &pi;/2. */
+    private static final double PI_OVER_2 = 0.5 * Math.PI;
+    /** &pi;/4. */
+    private static final double PI_OVER_4 = 0.25 * Math.PI;
     /** Natural logarithm of 2 (ln(2)). */
     private static final double LN_2 = Math.log(2);
-    /** {@code 1/2}. */
-    private static final double HALF = 0.5;
-    /** Base 10 logarithm of e divided by 2 (log10(e)/2). */
+    /** Base 10 logarithm of 10 divided by 2 (log10(e)/2). */
```

Comment:
This was one of the comments I fixed in the feature branch. Be careful what you cut and paste! It is the `Base 10 logarithm of e divided by 2`